site/en/guides/border-router/external-commissioning/prepare.md

```diff
@@ -61,7 +61,7 @@ OpenThread POSIX, using `ot-ctl`.
     was used in the operational dataset:
     ```
     $ cd ~/ot-br-posix/build/otbr/tools
-    $ ./pskc J01NME DEAD1111DEAD2222 OpenThreadGuide
+    $ ./pskc j01Nme DEAD1111DEAD2222 OpenThreadGuide
```

Comment:
@sarah-iot Does changing this affect any of the other guides?

Reply:
Yeah I think we should update globally for consistency.  We can merge this and fix the others separately.